bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2018.html

```diff
@@ -0,0 +1,253 @@
+  {% endblock %}
+
+  {% block report_body %}
+    <section class="section" id="government-user-data">
+      <h2 class="section-title" data-accordion-role="tab">{{ _('Government Demands for User Data') }}</h2>
+
+      <div class="content" data-accordion-role="tabpanel">
+        <p>
+          {{ _('In the Reporting Period, Mozilla received 1 Court Order and 1 Subpoena for user data.') }}
+        </p>
+
+        <table class="data-table">
+          <thead>
+            <tr>
+              <th scope="col">{{ _('Legal Processes') }}</th>
+              <th scope="col">{{ _('Received') }}</th>
+              <th scope="col">{{ _('Data Produced') }}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-search-warrant' }}">{{ _('Search Warrants') }}</a>
+              </th>
+              <td>0</td>
+              <td>{{ _('N/A') }}</td>
+            </tr>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-subpoena' }}">{{ _('Subpoenas') }}</a>
+              </th>
+              <td>1<sup><a href="#footnote-1">2</a></sup></td>
+              <td>{{ _('No') }}</td>
+            </tr>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-court-order' }}">{{ _('Court Orders') }}</a>
+              </th>
+              <td>1<sup><a href="#footnote-1">3</a></sup></td>
+              <td>{{ _('No') }}</td>
+            </tr>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-wiretap-order' }}">{{ _('Wiretap Orders') }}</a>
+              </th>
+              <td>0</td>
+              <td>{{ _('N/A') }}</td>
+            </tr>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-pen-register-order' }}">{{ _('Pen Register Orders') }}</a>
+              </th>
+              <td>0</td>
+              <td>{{ _('N/A') }}</td>
+            </tr>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-emergency-request' }}">{{ _('Emergency Requests') }}</a>
+              </th>
+              <td>0</td>
+              <td>{{ _('N/A') }}</td>
+            </tr>
+            <tr>
+              <th scope="row">
+                <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-natl-security-request' }}">{{ _('National Security Requests') }}</a> <sup><a href="#footnote-1">1</a></sup>
+              </th>
+              <td>0</td>
+              <td>{{ _('N/A') }}</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <aside class="footnotes">
+          <section id="footnote-1">
```

Comment:
Each footnote should be in its own section with its own ID, so the links can link directly to them separately. Look at the Jan-June 2017 report for an example (our only previous report with more than one footnote): https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2017.html#L87